plugins/modules/ntnx_object_stores_v2.py

```diff
@@ -0,0 +1,619 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Nutanix
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+module: ntnx_object_stores_v2
+short_description: Create, Update and Delete a Nutanix object store
+version_added: 2.2.0
+description:
+    - Create, Update and Delete a Nutanix object store.
+    - This module uses PC v4 APIs based SDKs
+options:
+    state:
+        description:
+            - State of the Nutanix object store, whether to create, update or delete.
+            - If C(state) is present, it will create or update the object store.
+            - If C(state) is set to C(present) and ext_id is not provided then it will create a new object store.
+            - If C(state) is set to C(present) and ext_id is provided then it will update the object store.
+            - If C(state) is absent, and ext_id is provided then it will delete the object store.
+            - Update object store is only supported for object stores in C(OBJECT_STORE_DEPLOYMENT_FAILED) state.
+            - If the object store is in C(OBJECT_STORE_DEPLOYMENT_FAILED) state, it can be updated to redeploy the object store.
+        choices: ['present', 'absent']
+        type: str
+        default: present
+    ext_id:
+        description:
+            - External ID of the object store.
+            - Used to update or delete the object store.
+        type: str
+        required: false
+    name:
+        description:
+            - Name of the object store.
+        type: str
+        required: false
+    metadata:
+        description:
+            - Metadata associated with the object store.
+        type: dict
+        required: false
+        suboptions:
+            owner_reference_id:
```

Comment:
mention if owner_reference_id is mandatory to assign a owner or not. As there is another parameter `owner_user_name` which might not be sufficient to assign owner

Reply:
Same for projects